src/Symfony/Component/Intl/Resources/bin/common.php

```diff
@@ -68,9 +68,10 @@ function get_icu_version_from_genrb($genrb)
     return $matches[1];
 }
 
-set_exception_handler(function (\Exception $exception) {
+set_exception_handler(function ($exception) {
```

Comment:
I would even typehint as Throwable directly here. This script is a maintainer-only script, so it is fine IMO (and its output is not meant to depend on the PHP version anyway)